docs/introduction.rst

```diff
@@ -11,106 +11,143 @@ or modifying files produced by other programs (e.g.,
-kept in the tree sequence after simplification. In contrast, individuals that are
-merely marked for retention can lose their nodes (as has happened to one of the nodes in
-this case). Furthermore, once all an individuals's nodes are lost, simplification
-removes the individual too, for efficiency reasons.
-
+OK, who and what exactly *is* left in the tree sequence after the unnecessary
+information has been removed? Here's the diagram above with some added terminology. Note
+that for simplicity, in this model the individuals that are currently alive are
+all in the final generation, but more generally, individuals alive at a particular time
+should be identified via the :meth:`.SlimTreeSequence.individuals_alive_at()` method (see
+:ref:`sec_extracting_individuals`).
 
 .. figure:: _static/pedigree2.png
    :scale: 40%
 
-The nodes within the final generation, together with those in the permanently
-remembered individuals are the *sample nodes* of the tree sequence.
-Saying that a chromosome is a *sample* implies that we have full information about
-its genealogy, and many operations on tree sequences act on the sample nodes
-by default (see the
-`tskit data model <https://tskit.readthedocs.io/en/latest/data-model.html#nodes-genomes-or-individuals>`_
-for more on this topic).
-The easiest way to pull out individuals alive at a particular time
-is with the :meth:`.SlimTreeSequence.individuals_alive_at()` method:
-see :ref:`sec_extracting_individuals` for more discussion.
-
-Here's a reminder of the terminology:
-
+In the recorded tree sequence, the alive individuals have all had their nodes marked as
+*sample nodes* which, as you may recall, means we have full ancestral information about
+them. As shown in the diagram this means the sample nodes, and the individuals
+containing them, are always present in the tree sequence (it's also possible to
+mark additional nodes as samples: see :ref:`sec_remembering_individuals`)
+
+In contrast to the individuals containing sample nodes, you can see that all the other
+circles, representing historical (i.e. dead) individuals, have gone, although sometimes
+their nodes remain. By default, only individuals with sample nodes are recorded in the
+tree sequence; that means ancestral nodes lose any information about which individuals
+they were in (see :ref:`sec_retaining_individuals` if you need to retain this information).
+
+As well as the actual individuals, many historical *nodes*  have been removed too,
+along with the lines linking them to the rest of the genealogy (in tree-sequence-speak the
+lines are known as "edges"). The deleted nodes are simply those that are not needed to
+reconstruct the relationships between the sample nodes. For example we remove nodes
+leading to a dead end (e.g. in individuals who had no offspring). Similarly, as time goes
+on, successive recombination events can gradually reduce the genetic contribution of
```

Comment:
it's genetic drift rather than recombination events that does this, strictly speaking

Reply:
ISWYM, but here I think it's a combination of both. What I'm trying to convey is that even if there are lines of ancestry in the diagram that go through a (coalescent) node, that node may be deleted because the line essentially "belongs to another tree". I'l try to rephrase.